docs/tutorial/progress-bar.md

```diff
@@ -1,35 +1,56 @@
 # Progress Bar in Taskbar (Windows, macOS, Unity)
 
-On Windows a taskbar button can be used to display a progress bar. This enables
-a window to provide progress information to the user without the user having to
-switch to the window itself.
+## Overview
 
-On macOS the progress bar will be displayed as a part of the dock icon.
+On Windows, a taskbar button can be used to display a progress bar. This enables
```

Comment:
```suggestion
On Windows, you can use a taskbar button to display a progress bar. This enables
```

I feel like we're using "You" a lot across the different guides, so consistent tone is probably useful here.